bam_collapsing.cwl

```diff
@@ -1,602 +1,817 @@
     outputSource:
-      - marianas_collapsing_second_pass_cwl/collapsed_fastq_1
+      - >-
+        fgbio_collect_duplex_seq_metrics_1_2_0/fgbio_collect_duplex_seq_metrics_duplex_yield_metrics
     type: File
-    'sbg:x': 1141.263671875
-    'sbg:y': 2041.484375
-  - id: pileup_without_duplicates
+    'sbg:x': 1624.7490234375
+    'sbg:y': 2282.859375
+  - id: fgbio_collect_duplex_seq_metrics_duplex_umi_counts
     outputSource:
-      - waltz_pileupmetrics/pileup_without_duplicates
+      - >-
+        fgbio_collect_duplex_seq_metrics_1_2_0/fgbio_collect_duplex_seq_metrics_duplex_umi_counts
     type: File
-    'sbg:x': 686.4918212890625
-    'sbg:y': 1282.2890625
-  - id: intervals_without_duplicates
+    'sbg:x': 1624.7490234375
+    'sbg:y': 2389.59375
+  - id: fgbio_collect_duplex_seq_metrics_duplex_qc
     outputSource:
-      - waltz_pileupmetrics/intervals_without_duplicates
+      - >-
+        fgbio_collect_duplex_seq_metrics_1_2_0/fgbio_collect_duplex_seq_metrics_duplex_qc
     type: File
-    'sbg:x': 686.4918212890625
-    'sbg:y': 1919.0859375
-  - id: intervals
+    'sbg:x': 1624.7490234375
+    'sbg:y': 2496.328125
+  - id: fgbio_collect_duplex_seq_metrics_duplex_family_size
     outputSource:
-      - waltz_pileupmetrics/intervals
+      - >-
+        fgbio_collect_duplex_seq_metrics_1_2_0/fgbio_collect_duplex_seq_metrics_duplex_family_size
     type: File
-    'sbg:x': 686.4918212890625
-    'sbg:y': 2025.7734375
-  - id: unfiltered-bam
+    'sbg:x': 1624.7490234375
+    'sbg:y': 2603.0625
+  - id: gatk_sam_to_fastq_unpaired_fastq
     outputSource:
-      - abra_fx/abra_fx_bam
-    type: File
-    secondaryFiles:
-      - ^.bai
-    'sbg:x': 2415.73974609375
-    'sbg:y': 1479.953125
-  - id: output_file
-    outputSource:
-      - abra_fx/output_file
+      - gatk_sam_to_fastq_4_1_8_0/gatk_sam_to_fastq_unpaired_fastq
     type: File?
-    label: indel_realign_targets
-    'sbg:x': 2415.73974609375
-    'sbg:y': 1707.3984375
-  - id: first_pass_insertions
-    outputSource:
-      - marianas_collapsing_first_pass_cwl/first_pass_insertions
-    type: File
-    'sbg:x': 1141.263671875
-    'sbg:y': 1828.109375
-  - id: alt_allele_file
+    'sbg:x': 2182.387451171875
+    'sbg:y': 1392.734375
+  - id: gatk_sam_to_fastq_second_end_fastq
     outputSource:
-      - marianas_collapsing_first_pass_cwl/alt_allele_file
-    type: File
-    'sbg:x': 1141.263671875
-    'sbg:y': 2148.171875
-  - id: first_pass_output_dir
-    outputSource:
-      - marianas_collapsing_first_pass_cwl/first_pass_output_dir
-    type: Directory
-    'sbg:x': 1141.263671875
-    'sbg:y': 1721.421875
-  - id: gzip_read1
-    outputSource:
-      - gzip_Read1/output_file
-    type: File
-    'sbg:x': 1386.576171875
-    'sbg:y': 1509.640625
-  - id: gzip_read2
+      - gatk_sam_to_fastq_4_1_8_0/gatk_sam_to_fastq_second_end_fastq
+    type: File?
+    'sbg:x': 2182.387451171875
+    'sbg:y': 1499.46875
+  - id: gatk_sam_to_fastq_fastq
```

Comment:
Do we need to save the fastqs from GATK samtofastq? maybe we want to check that the UMIs are correctly formatted in the read header? or maybe we don't need to check them

Reply:
Sorry, just remembered these would be the collapsed fastqs, which probably would make sense to save